third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs

```diff
@@ -36,7 +39,10 @@ pub struct FunctionGenerator<'a> {
     /// A map from a temporary to information associated with it.
     temps: BTreeMap<TempIndex, TempInfo>,
     /// The value stack, represented by the temporaries which are located on it.
-    stack: Vec<TempIndex>,
+    /// Each temporary is paired with a bool, indicating whether the value was copied onto
+    /// the stack from a local.
```

Comment:
I have changed this a bit from previous PR to clarify. This information is not available in `TempInfo`, which gives the mapping from a temp to a local.